drivers/storage/portworx/status.go

```diff
@@ -270,6 +304,43 @@ func (p *portworx) updateStorageNodeStatus(
 		Type:   corev1alpha1.NodeStateCondition,
 		Status: mapNodeStatus(node.Status),
 	}
+
+	var (
+		totalSizeInBytes, usedSizeInBytes int64
+	)
+
+	for _, pool := range node.Pools {
+		totalSizeInBytes += int64(pool.TotalSize)
+		usedSizeInBytes += int64(pool.Used)
+	}
+	storageNode.Status.Storage = corev1alpha1.StorageStatus{
+		TotalSize: *resource.NewQuantity(totalSizeInBytes, resource.BinarySI),
+		UsedSize:  *resource.NewQuantity(usedSizeInBytes, resource.BinarySI),
+	}
+
+	kvdbEntry, present := kvdbNodeMap[storageNode.Status.NodeUID]
+	if present && kvdbEntry != nil {
+		nodeKVDBCondition := &corev1alpha1.NodeCondition{
+			Type:   corev1alpha1.NodeKVDBCondition,
+			Status: mapKVDBState(kvdbEntry.State),
+			Message: fmt.Sprintf("node is kvdb %s listening on %s",
```

Comment:
the node.Type is an int, so this message will be confusing for the user. May be change this to
`node is running kvdb and is listening on port %s`

Reply:
The %s in the message is `mapKVDBNodeType(kvdbEntry.Type)`. So it will print either "leader" or "member" in string format